src/design-tokens/tier-3-component/focus-ring.json

```diff
@@ -10,6 +10,18 @@
             "value": "{eds.color.neutral.white}"
           }
         }
+      },
```

Comment:
for later: perhaps these should be part of the forms.json, as it will apply to all form elements (and maybe form should actually be the individual form elements?) Just stirring the pot but food for thought.

Reply:
Yea there's some discussion to be had around nomenclature around tokens and possibly condensing the amount of possibilities